cloud/azure/postgresql/detectors-postgresql.tf

```diff
@@ -0,0 +1,154 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Add `threshold` function & use `lasting` function instead of aggregation function for PostgreSQL detectors.